contractcourt/htlc_outgoing_contest_resolver.go

```diff
@@ -134,18 +149,23 @@ func (h *htlcOutgoingContestResolver) Resolve() (ContractResolver, error) {
 				return &h.htlcTimeoutResolver, nil
```

Comment:
One thing to be aware of is that a retribution tx can also still happen after expiry. It is then to be handled in the timeout resolver. But to me that looks like scope for a follow up pr though.